src/chrome/content/rules/Sohu.com-mixedcontent.xml

```diff
@@ -0,0 +1,28 @@
+<!--
+	Other Sohu rulesets:
+		- Sohu.com.xml
+		- Sohu.com.cn.xml
+		- Sohu.net.xml
+		- SohuCS.com.xml
+		- SohuSCE.com.xml
```

Comment:
Please just use `For other Sohu coverage, see Sohu.com.xml` to avoid additional maintenance.